worker/caasfirewallersidecar/worker_test.go

```diff
@@ -27,14 +30,20 @@ type workerSuite struct {
 
 	firewallerAPI *mocks.MockCAASFirewallerAPI
 	lifeGetter    *mocks.MockLifeGetter
-	logger        *mocks.MockLogger
 	broker        *mocks.MockCAASBroker
 
 	applicationChanges chan []string
 
 	appsWatcher watcher.StringsWatcher
 }
 
+type noopLogger struct{}
```

Comment:
github.com/juju/juju/testing has a NoopLogger

Reply:
I had actually switched to using `loggo.GetLogger("test")` like a lot of the other tests (doesn't need to be a no-op), and forgotten to get rid of this type. Removed.